packages/web-components/src/card/card.ts

```diff
@@ -0,0 +1,207 @@
+import { attr, css, ElementStyles, observable } from '@microsoft/fast-element';
+import { FASTCard, StartEndOptions } from '@microsoft/fast-foundation';
+import { keyEnter, keySpace } from '@microsoft/fast-web-utilities';
+import { CardAppearance, CardControlSize, CardOrientation } from './card.options.js';
+
+/**
+ * Card configuration options
+ * @public
+ */
+export type CardOptions = StartEndOptions<Card>;
+
+/**
+ * @class Card component
+ *
+ * @remarks
+ * This class extends the FASTCard. a flexible content container
+ */
+export class Card extends FASTCard {
+  public connectedCallback(): void {
+    super.connectedCallback();
+    this.setInteractive();
+    this.updateComputedStylesheet();
+  }
+
+  public disconnectedCallback(): void {
+    super.disconnectedCallback();
+  }
+
+  /**
+   * Stores the computed stylesheet for the card
+   */
+  private computedStylesheet?: ElementStyles;
+
+  /**
+   * @property orientation;
+   * @default filled
+   * @remarks
+   * Determines the orientation of the card
+   */
+  @attr
+  public appearance?: CardAppearance;
+
+  /**
+   * @property orientation;
+   * @default vertical
+   * @remarks
+   * Determines the orientation of the card
+   */
+  @attr
+  public orientation?: CardOrientation;
+
+  /**
+   * @property controlSize
+   * @default medium
+   * @remarks
+   * Determines the size of the card
+   */
+  @attr({ attribute: 'control-size' })
+  public controlSize?: CardControlSize;
+
+  /**
+   * @property interactive
+   * @default false
+   * @remarks
+   * Determines whether card is interactable
+   */
+  @observable
+  @attr({ mode: 'boolean' })
+  public interactive: boolean = false;
+
+  /**
+   * @property disabled
+   * @default false
+   * @remarks
+   * Determines disabled state of card
+   */
+  @attr({ mode: 'boolean' })
+  public disabled: boolean = false;
+
+  /**
+   * @property selected
+   * @default false
+   * @remarks
+   * Determines selected state of card
+   */
+  @observable
+  @attr({ mode: 'boolean' })
+  public selected: boolean = false;
+
+  /**
+   * @method sizeChanged
+   * @remarks
+   * Updates the computed stylesheet when the size of the card changes
+   */
+  public sizeChanged(prev: boolean | undefined, next: boolean): void {
+    this.updateComputedStylesheet();
+  }
```

Comment:
There isn't a `size` attribute from what I can see in this class, you've named it `controlSize`...